deleted.txt

```diff
@@ -90,4 +90,5 @@ f94bad2d-0c76-4e8f-a0d3-970810093c6c # The Age; replaced by generic Fairfax Aust
 d8873d23-d874-4b62-b081-1db12ff5a5de # ILO Labordoc; covered by EM and Primo now
 ac721d14-f081-4613-91e8-255c22ad05eb # IASSIST Quarterly
 62c0e36a-ee2f-4aa0-b111-5e2cbd7bb5ba # MetaPress
+a77690cf-c5d1-8fc4-110f-d1fc765dcf88 # ProQuest Legacy.js, see issue #660
 fcfcfe9c-f6dd-48c6-aef9-61adbba31a4e # National Library of Australia - Australian Newspapers covered by Trove now
```

Comment:
It is IMO better to have the ordering as it really appeared in the deletions, i.e. move the ProQuest Legacy translator to the bottom.